adoc/Private-Registry-Installation.adoc

```diff
@@ -62,27 +68,27 @@ export KUBECONFIG=<full path to kubeconfig.yaml>
 The default configuration provided with the {spr} helm chart is not suited for production use!
 
 A separate YAML file (referred to as the `harbor-values.yaml` file in the following sections) needs to be populated with customized configuration values to be used during installation.
-The exact configuration options that can be customized, as well as the values that they can take are covered in detail in the next installation steps.
+The exact configuration options that can be customized, as well as the values that they can take, are covered in detail in the next installation steps.
```

Comment:
Following the style guide, don't we usually avoid "as well as" and use "and" instead?

The exact configuration options that can be customized, and the values that they can take, ...